### ISSUE TITLE
About the ground truth of Holopix50K

### ISSUE THREAD

**Issue by gangweiX** [megvii-research/CREStereo#4]
您好，感谢您的CREStereo，我想问一下关于Holopix50K上的预测结果，模型有没有在Holopix50K上面进行预训练，如果预训练的话，这个数据集的GT该如何获得呢

**Comment by DY-ATL** [comment]
There is no real GT for Holopix50K. Holopix50K is usually for unsupervised learning, or we can obtain pseudo labels of it using other disparity estimation methods.
I wonder if Holopix50K is used for pretraining, too.

**Comment by JackLee396** [comment]
The training set does not contain Holopix50K. We only use it to validate the model performance.
